modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -27,8 +27,8 @@ definition {
 					"recipientType": "email",
 					"recipients": [
 						{
-							"cc": "",
-							"bcc": "",
+							"cc": "${cc}",
+							"bcc": "${bcc}",
```

Comment:
Please squash this into the original commit. It's better to do it right the first time instead of fixing your mistake with a second commit. Don't forget to use present tense for the commit message!